src/renderers/common/XRManager.js

```diff
@@ -596,7 +596,22 @@ class XRManager extends EventDispatcher {
 
 	}
 
-	createQuadLayer( width, height, translation, quaternion, pixelwidth, pixelheight, rendercall, attributes = [] ) {
```

Comment:
Looking at how `attributes` is used in the code, the parameter should be an object, not an array.